### PR TITLE
Prevent route being included in intial state

### PR DESCRIPTION
**Pull request by ellsclytn**
Previously, the route (a multi-megabyte array of objects) was being included in both the HTML (in initial state) as well as the JSON. Using the `defer` hook restricts the route fetch to happen only in the client JS, meaning it won't be included in the initial state of a static build

### CODE REVIEW

**Comment by ellsclytn** [comment]
Ignore Buildkite for now. I'll raise a separate PR to get it going (I've also been working on it)

File: source/store/tourTracker/index.js [hunk 1]
```diff
@@ -1,8 +1,12 @@
 // @flow
-import { reducer } from 'tour-tracker'
 import styles from './styles'
 import points from './points.json'
 
+// Constants
+const c = {
```
Comment (kizza) [inline source/store/tourTracker/index.js]: Is this a thing we do?  Have a `c` object with its keys containing constants? (haven't seen that pattern, perhaps it's certainly a thing)

**Comment by ellsclytn** [comment]
I stole it out of other PS sites. Admittedly, it's more useful when there's
multiple constants, but I'm just following a De facto convention here.

File: source/routes/Home/index.js [hunk 2]
```diff
@@ -49,6 +49,13 @@ const hooks = {
         end
       }))
     })
+  },
+  defer ({
+    dispatch,
+    state
+  }) {
+    return Promise.resolve()
```
Comment (kizza) [inline source/routes/Home/index.js]: Can be explicit return? (does anyone care?)

File: source/store/tourTracker/index.js (hunk 1 above)
Comment (kizza) [inline source/store/tourTracker/index.js]: Sure... as you wish, does seem odd to call the object `c` and describe in a comment above that it really means `constants` (lol).  Variable names are hard :)

File: source/routes/Home/index.js (hunk 2 above)
Comment (ellsclytn) [inline source/routes/Home/index.js]: It totally can.

**Pull request merged**